frontend/src/app/profile/profile-editor/profile-editor.component.ts

```diff
@@ -62,6 +63,15 @@ export class ProfileEditorComponent implements OnInit {
     });
   }
 
+  displayToken(): void {
+    this.token = String(localStorage.getItem('bearerToken'));
+  }
+
+  copyToken(): void {
+    navigator.clipboard.writeText(String(localStorage.getItem('bearerToken')));
+    this.snackBar.open('Token Copied', '', { duration: 2000 });
```

Comment:
Love seeing the snackBar opening up! We'll be able to reference the loaded token rather than reading from `localStorage` again.